br/tests/br_other/run.sh

```diff
@@ -52,7 +52,7 @@ run_br debug backupmeta validate -s "local://$TEST_DIR/$DB" --offset 100
 run_br validate checksum -s "local://$TEST_DIR/$DB"
 
 # Test validate checksum
-for sst in $TEST_DIR/$DB/*.sst; do
+for sst in $TEST_DIR/$DB/*/*.sst; do
```

Comment:
it's from https://github.com/pingcap/tidb/pull/36411/files#diff-ce8d327b81b0263e3df527152c8d36f5936ecae9b1d08c6849ea640a543391f8R55

since is quite small, it's merged into this pr